### PR TITLE
fixed padding on remove confirmation message and buttons, fixes #220

### LINKED ISSUE #220: Add padding to Course deletion buttons
On the alert message for Course delete, add padding and spacing to the "Delete" and "cancel" buttons to improve presentation and to help avoid accidental deletions when someone tries to click the "cancel" button.

### PR DESCRIPTION
**Pull request by jenzweben**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/ilios/frontend/pull/225" target="_blank">ReviewNinja

File: app/styles/components/_resultslist.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .resultslist {
-  $remove-background-color: lighten($ilios-red, 40%);
+  // $remove-background-color: lighten($ilios-red, 40%);
```
Comment (jrjohnson) [inline app/styles/components/_resultslist.scss]: Please remove this line instead of commenting it out.

**Comment by jenzweben** [comment]
fixed line 141

**Comment by jrjohnson** [comment]
Replaced by #238

**Pull request closed without merging**